types/underscore/underscore-tests.ts

```diff
@@ -165,372 +165,423 @@ declare const level2UnionList: _.List<_.List<string | number>>;
 declare const tupleList: _.List<[string, number]>;
 declare const maybeFunction: (() => void) | undefined;
 
+// concrete example types
+declare const manyParameters: (a: string, b: number, c: boolean, d: string, e: number, f: string) => string;
+const stooges = [{ name: 'moe', age: 40 }, { name: 'larry', age: 50 }, { name: 'curly', age: 60 }];
```

Comment:
I feel like I can't call it `stooges` unless all three are in there 😛 

That being said, it's probably not necessary to call this `stooges`, and that may be a reference that many folks may not be familiar with (I myself have only a vague awareness of the existence of the three stooges). It would probably be better to change this to something more generic like `declare const people: { name: string; age: number; }[];`.

Reply:
I made the change described above; let me know if you'd prefer to have me change it back. I suppose it can be fun and even a little energizing to find a light-hearted reference to something in test code rather than having it all be bland boring blah, and this change does take that away.

Reply:
Yeah, it would be nice to keep the `stooges` name (even though I don't actually know the reference myself). Your "excuse" for keeping it a literal value is fine by me.

Reply:
Reverted 49c0a79 as 1d7aa13 😄